SPECS-EXTENDED/gdisk/gdisk.spec

```diff
@@ -0,0 +1,250 @@
+Summary:        An fdisk-like partitioning tool for GPT disks
+Name:           gdisk
+Version:        1.0.9
+Release:        3%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.rodsbooks.com/gdisk/
+Source0:        https://downloads.sourceforge.net/gptfdisk/gptfdisk-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         gdisk-libuuid.patch
+BuildRequires:  gcc-c++
+BuildRequires:  libuuid-devel
+BuildRequires:  make
+BuildRequires:  ncurses-devel
+BuildRequires:  popt-devel
+
+%description
+An fdisk-like partitioning tool for GPT disks. GPT fdisk features a
+command-line interface, fairly direct manipulation of partition table
+structures, recovery tools to help you deal with corrupt partition
+tables, and the ability to convert MBR disks to GPT format.
+
+%prep
+%setup -q -n gptfdisk-%{version}
+%patch0 -p1 -b .orig
+
+%build
+make CXXFLAGS="%{optflags} -D_FILE_OFFSET_BITS=64" LDFLAGS="%{build_ldflags}"
+
+%install
+for f in gdisk sgdisk cgdisk fixparts ; do
+    install -D -p -m 0755 $f %{buildroot}%{_sbindir}/$f
+    install -D -p -m 0644 $f.8 %{buildroot}%{_mandir}/man8/$f.8
+done
+
+%check
+make test
```

Comment:
Have we verified the tests pass? The linked buddy builds ran with disabled tests, so it's hard to tell.